src/blocks/template-everything/render.js

```diff
@@ -0,0 +1,47 @@
+import classnames from 'classnames';
+import React from 'react';
```

Comment:
lets avoid importing from react
```suggestion
import { memo } from '@wordpress/element';
```